### PR TITLE
🦶 Builtin footgun prevention

### PR DESCRIPTION
**Pull request by rowanseymour**
Adds #915

### CODE REVIEW

File: flows/actions/testdata/send_broadcast.json [hunk 1]
```diff
@@ -72,6 +72,26 @@
...
+                "tel:+1234567890"
+            ],
+            "text": "Hi there!"
+        },
+        "events": [
+            {
+                "type": "error",
+                "created_on": "2018-10-18T14:20:30.000123456Z",
+                "step_uuid": "59d74b86-3e2f-4a93-aece-b05d2fdcde0c",
+                "text": "can't send broadcasts during batch starts"
```
Comment (rowanseymour) [inline flows/actions/testdata/send_broadcast.json:91]: is this maybe too strict?

Comment (nicpottier) [reply flows/actions/testdata/send_broadcast.json:91]: sending broadcasts? It is the original footgun so no, I don't think so.

Comment (rowanseymour) [reply flows/actions/testdata/send_broadcast.json:91]: I thought maybe there's a valid case where your sending to one person.. like you start a bunch of people in a flow that messages their supervisor... so I was wondering if this should allow a broadcast to 1 URN or contact

Comment (nicpottier) [reply flows/actions/testdata/send_broadcast.json:91]: Hah, was literally just looking at that myself. But yes, should be consistent.

**Pull request merged**